component/src/main/java/org/wso2/extension/siddhi/store/cassandra/CassandraEventTable.java

```diff
@@ -0,0 +1,1152 @@
+        description = "This extension assigns data sources and connection instructions to event tables. It also " +
+                "implements read-write operations on connected datasource.",
+        parameters = {
+                @Parameter(name = "cassandra.host",
+                        description = "Host that is used to get connected in to the cassandra keyspace.",
+                        type = {DataType.STRING},
+                        defaultValue = "localhost"),
+                @Parameter(name = "column.family",
+                        description = "The name with which the event table should be persisted in the store. If no " +
+                                "name is specified via this parameter, the event table is persisted with the same " +
+                                "name as the Siddhi table.",
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "The table name defined in the Siddhi Application query."),
+                @Parameter(name = "client.port",
+                        description = "Client port that is used to get connected with the client store. If no " +
+                                "name is specified via this parameter, the default port will be taken.",
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "9042"),
+                @Parameter(name = "keyspace",
+                        description = "User need to give the keyspace that the data is persisted. " +
+                                "Keyspace name is specified via this parameter.",
+                        type = {DataType.STRING},
+                        defaultValue = "'cassandraTestTable'"),
+                @Parameter(name = "username",
+                        description = "Through user name user can specify the relevent username " +
+                                "that is used to log in to the cassandra keyspace .",
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "null"
+                ),
+                @Parameter(name = "password",
+                        description = "Through password user can specify the relevent password " +
+                                "that is used to log in to the cassandra keyspace .",
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "null"
+                )
+        },
+        examples = {
+                @Example(
+                        syntax = "define stream StockStream (symbol string, price float, volume long); \n" +
+                                "@Store(type=\"cassandra\", table.name=\"StockTable\",keyspace=\"AnalyticsFamily\"," +
+                                "username=\"cassandra\",password=\"cassandra\",cassandra.host=\"localhost\")" +
```

Comment:
Normally we used single quotations marks rather than the double quotation marks when specifying the value of parameters. (ex: username='cassandara', ....)

Reply:
@tharindulak has this been addressed?